tvtk/pyface/ui/qt4/QVTKRenderWindowInteractor.py

```diff
@@ -787,4 +794,5 @@ def _qt_key_to_key_sym(key):
 
 
 if __name__ == "__main__":
+    print(PyQtImpl)
```

Comment:
Did you forget to delete debug print output?
```suggestion
```

Reply:
This is part of the original upstream version and is in the `__main__` block so I think it is fine as it is convenient when debugging without having to edit the source.

Reply:
Got it thanks!